src/frontend/js/clusive-prefs-panel.js

```diff
@@ -71,32 +71,32 @@
 
         if ($.isEmptyObject(model)) {
             var resetURL = directModel.resetURL;
-            $.get(resetURL, function(data) {
-                console.debug(resetURL, data);
-            });
-        } else {
-            var getURL = directModel.getURL;
-
-            $.get(getURL, function(currentPrefs) {
-                fluid.each(fluid.get(model, 'preferences'), function(prefVal, prefKey) {
-                    // Implicit conversion of numbers as strings to compare with numbers
-                    if (currentPrefs[prefKey] != prefVal) {
-                        var setURL = fluid.stringTemplate(directModel.setURL, {
-                            prefKey: prefKey,
-                            prefVal: prefVal
-                        });
-                        $.get(setURL, function(data) {
-                            console.debug(setURL, data);
-                        });
-                    } else {
-                        var message = fluid.stringTemplate('%prefKey already stored at value \'%prefVal\', not making save request', {
-                            prefVal: prefVal,
-                            prefKey:prefKey
-                        });
-                        console.debug(message);
-                    }
-                });
-            });
+            $.ajax({
+                type: "POST",
+                url: resetURL,
+                headers: {
+                    'X-CSRFToken': DJANGO_CSRF_TOKEN
+                },
+                data: JSON.stringify({adopt: 'default'}),
+                success: function (data) {
+                    console.log("reset preferences to default", data);
```

Comment:
This was happening by "coincidence" (reset default on client is same as reset default from server side); I've now made it explicit. It could be improved to reduce the need to hit the server twice; I'll create a JIRA for this.